### DISCUSSION TITLE
Check if `null`

### DISCUSSION THREAD

**Discussion by vitoyucepi** [savonet/liquidsoap#1900] [Q&A] [answered]
I've found two ways to check if variable is `null`.

1.  `null.defined: test whether a value is null or not` https://github.com/savonet/liquidsoap/blob/801625b718db3d18e9025fa28bf602a0ee2c4f4f/libs/metadata.liq#L149

1.  compare with `null()` https://github.com/savonet/liquidsoap/blob/801625b718db3d18e9025fa28bf602a0ee2c4f4f/tests/streams/say.liq#L6

Which method is better?

**Comment by smimram** [comment, marked as answer]
Both ways should be equivalent.
